lib/web_ui/lib/src/engine/dom_renderer.dart

```diff
@@ -194,11 +172,11 @@ class DomRenderer {
     js_util.setProperty(element, name, value);
   }
 
-  void setElementStyle(html.Element element, String name, String value) {
+  void setElementStyle(html.Element? element, String name, String? value) {
```

Comment:
html.Element should be non nullable? If it is causing too much change at call sites, consider adding TODO.

Reply:
Done.